README.md

```diff
@@ -53,37 +73,39 @@ Silk uses [PostgreSQL](https://www.postgresql.org/) for its backend database sto
    - ```
      <UserSecretsId>VelvetThePanda-SilkBot</UserSecretsId>
      ```
-2. The location of the ```secrets``` file is different between Operating Systems, but in a system-protected user profile folder on your computer.
+2. The location of the ```secrets.json``` file is different between Operating Systems, but in a system-protected user profile folder on your computer.
```

Comment:
Shouldn't this be inline codeblock (\`\`) instead of a full codeblock (\`\`\`\`\`\`)?

Reply:
Yup will change those that are like that.